src/modules/partners/partner_service.ts

```diff
@@ -0,0 +1,16 @@
+import { perPage } from '../../helpers/paginate'
+import { Partner as Entity } from './partner_entity'
+import { Partner as Repository } from './partner_repository'
+
+export namespace Partner {
+  export const findAll = async (requestQuery: Entity.RequestQuery) => {
+    const query = Repository.Partners()
+      .select('id', 'name', 'total_village', 'logo', 'created_at')
```

Comment:
I am confused if using repository this sql logic save in repository not in service

Reply:
updated ccdad35